website/docs/d/nsxt_edgegateway.html.markdown

```diff
@@ -10,29 +10,48 @@ description: |-
 
 Provides a VMware Cloud Director NSX-T edge gateway data source. This can be used to read NSX-T edge gateway configurations.
 
--> **Note:** This data source uses new VMware Cloud Director
-[OpenAPI](https://code.vmware.com/docs/11982/getting-started-with-vmware-cloud-director-openapi) and
-requires at least VCD *10.1.1+* and NSX-T *3.0+*.
-
 Supported in provider *v3.1+*.
 
-## Example Usage 
+## Example Usage (NSX-T Edge Gateway belonging to VDC group)
 
 ```hcl
+data "vcd_vdc_group" "group1" {
+  name = "existing-group"
+}
+
 data "vcd_nsxt_edgegateway" "t1" {
-  org  = "myorg"
-  vdc  = "my-nsxt-vdc"
-  name = "nsxt-edge-gateway"
+  org      = "myorg"
+  owner_id = data.vcd_vdc_group.group1.id
+  name     = "nsxt-edge-gateway"
 }
 ```
 
+## Example Usage (NSX-T Edge Gateway belonging to VDC)
+
+```hcl
+data "vcd_org_vdc" "vdc1" {
+  name = "existing-vdc"
+}
+
+data "vcd_nsxt_edgegateway" "t1" {
+  org      = "myorg"
+  owner_id = data.vcd_org_vdc.vdc1.id
+  name     = "nsxt-edge-gateway"
+}
+```
 
 ## Argument Reference
 
 The following arguments are supported:
 
-* `org` - (Optional) The name of organization to which the edge gatweway belongs. Optional if defined at provider level.
-* `vdc` - (Optional) The name of VDC that owns the edge gateway. Optional if defined at provider level.
+* `org` - (Optional) The name of organization to which the NSX-T Edge Gateway belongs. Optional if
+  defined at provider level.
+* `vdc` - (Optional)  **Deprecated** - please use `owner_id` field. The name of VDC that owns the
+  NSX-T Edge Gateway. Optional if defined at provider level.
+* `owner_id` - (Optional, *v3.6+*,*VCD 10.2+*) **Replaces** `owner_id`The ID of VDC group that this
```

Comment:
We state that `vdc` is Deprecated and use `owner_id`, but then in `owner_id` we say that it is only for VDC Group. How can someone replace `vdc` with `owner_id` for VDC (not Group)? This is my main feedback for this review scan. In the field descriptions in files above we have the same uncertainty.

Also, missing space:

```suggestion
* `owner_id` - (Optional, *v3.6+*,*VCD 10.2+*) **Replaces** `owner_id` The ID of VDC group that this
```

Reply:
Sorry. This is a nonsense I wrote during all the iterations I made and some details got messed up. It obviously replaces `vdc` field and supports both VDC and VDC Groups.
I have reviewed whole datasource file to cleanup things.